lib/tasks/export_tsv.rake

```diff
@@ -0,0 +1,4 @@
+desc "export a tsv summarising the most recent data in the database"
+task export_tsv: :environment do
+  ExportPlacesWithVotes.new.call
+end
```

Comment:
I suggest writing a test that mocks `ExportPlacesWithVotes#call` and checks it's called when the task is run.